libs/couch2pg/migrations/001.do.2318-standardise-schema.sql

```diff
@@ -0,0 +1,19 @@
+-- get everyone to baseline what existed before we added migrations
+CREATE TABLE IF NOT EXISTS couchdb (doc jsonb);
+
+-- IF NOT EXISTS doesn't exist (!) in Postgres 9.4, so do this silliness instead
+DO $$
+BEGIN
+
+IF NOT EXISTS (SELECT 1 FROM pg_class c WHERE c.relname = 'couchdb_doc_uuid') THEN
```

Comment:
We could probably abstract these three lines into a PG Function called `IF_INDEX_NOT_EXISTS()`, which would then be available to these three index commands.

I think that might actually be the way to go in the future because we might need to make some dynamic indices based on discovered form names if we keep XML parsing separate. 

https://github.com/medic/medic-webapp/issues/2232